src/site/markdown/server/manage-webadmin.md

```diff
@@ -967,6 +980,20 @@ The scheduled task will have the following type `RecomputeAllPreviewsTask` and t
 }
 ```
 
+
+An admin can specify the concurrency that should be used when running the task:
+
+ - `mailboxParallelism` for each user specify how many mailboxes should be processed concurrently
+ - `messageParallelism`for each mailbox specify how many messages should be processed concurrently
```

Comment:
for example what happens here if the admin provide the `userParallelism` parameter?